cli/cmd/inject.go

```diff
@@ -346,19 +359,49 @@ func InjectYAML(in io.Reader, out io.Writer, options *injectOptions) error {
 			return err
 		}
 
-		result, err := injectResource(bytes, options)
+		report := injectReport{}
+		result, err := injectResource(bytes, options, &report)
 		if err != nil {
 			return err
 		}
 
 		out.Write(result)
 		out.Write([]byte("---\n"))
+
+		injectReports = append(injectReports, report)
+	}
+
+	// validate whether any yaml documents were injected
+	injected := 0
+	hostNetwork := 0
+	unsupportedResource := 0
+
+	for _, r := range injectReports {
+		if !r.hostNetwork && !r.unsupportedResource {
+			injected++
+		} else if r.hostNetwork {
+			hostNetwork++
+		} else if r.unsupportedResource {
+			unsupportedResource++
+		}
+	}
+
+	if injected > 0 {
+		report.Write([]byte(fmt.Sprintf("%s %d of %d YAML document(s) injected\n", okStatus, injected, len(injectReports))))
```

Comment:
if everything went fine, should we print this at all? another way to put it: do we want to communicate to the user the number of documents that were injected?